### DISCUSSION TITLE
Rename directory 'current_release'

### DISCUSSION THREAD

**Discussion by jgadsden** [OWASP/www-project-developer-guide#2] [Q&A] [answered]
@hblankenship I think you mentioned this when we were talking at AppSec Dublin, but it would be good to rename directory 'current_release'? Would something like 'draft' or 'work-in-progress' be suitable?

@alezza , @Shruti-s-kulkarni and Vandana what are your thoughts?

**Comment by jgadsden** [comment]
@hblankenship how about we take a leaf out of Docker's book and name it 'latest'?
The [Security Culture project](https://github.com/OWASP/www-project-security-culture) uses 'stable' but I am not sure if that is as good as 'latest'

**Reply by hblankenship** [reply]
@jgadsden  Does 'latest' imply released? If so, then _draft (from Jekyll blogs -heh heh) might be a better option?

**Reply by jgadsden** [reply]
OK, sounds good - lets go with `_draft`

**Comment by jgadsden** [comment, marked as answer]
I have applied the rename from `current_release` to `_draft` in commit f22eec59ef296565b5fc7011684503510db45dda
